src/client.js

```diff
@@ -142,11 +142,26 @@ export default class Client {
       return baseQueryPromise;
     }
 
+    const page = params.page || {};
+
+    // We intentionally drop passed analytics tags here so that we don't get
+    // double counted search analytics in the dashboard from disjunctive
+    // calls
+    const analytics = params.analytics || {};
+    analytics.tags = ["Facet-Only"];
```

Comment:
Seems a bit heavy handed to hardcode this value. Possible to use a user provided value first, then fall back to this value if one isn't supplied?

Reply:
Yeah that's a great point. I'll add in a override option and keep this as the default.